doc/WebRTC.xml

```diff
@@ -48,17 +48,23 @@
         </author>
         <revremark>First release.</revremark>
       </revision>
+      <revision>
+      <revnumber>25.06</revnumber>
+        <date>June, 2025</date>
+        <author>
+          <personname>Tomasz Zajac</personname>
+        </author>
+        <revremark>Added data channels.</revremark>
+      </revision>
     </revhistory>
   </info>
   <chapter xml:id="chapter_txt_v12_v5b">
     <title>Scope </title>
     <para>This document defines how WebRTC and related protocols are to be used for ONVIF clients
       and devices to establish a peer-to-peer connection between a client and a device using a
       signaling server.</para>
-    <para> Sending event and analytics metadata as well as PTZ and other commands via WebRTC
-      datachannels is outside of the scope of this version of the specifcation. Similar future
-      versions of the specification may address a directory protocol to query organizations,
-      devices and profiles. </para>
+    <para>Sending PTZ and other commands via WebRTC datachannels is outside of the scope of this 

```

Comment:
I know that there was discussion on the Uplink allowing PTZ communication via SOAP requests across the HTTP/2 Backchannel or WebSocket. In this context a Client and Device are making a Peer To Peer connection via WebRTC and that device  may be a PTZ Camera and should this spec not allow the client to perform PTZ over the WebRTC backchannel.

Is there reasons why this is not updated as part of the Spec or are you only focusing on Metadata?

Reply:
My initial intention was to include SOAP commands over data channel as well, however during last F2F meeting there was no interest from the client side to standardize it. We can revisit this again and if there is any vendor wishing to prototype this on the client side, I will add it.